lib/GlpiLocalesExtension.php

```diff
@@ -61,4 +61,21 @@ public function getFunctions() {
    public function getName() {
       return 'glpi_locales_extension';
    }
+
+   public function getFilters()
+   {
+      return array(
+         new \Twig_SimpleFilter('fileSize', array($this, 'fileSizeFilter')),
+      );
+   }
+
+   /**
+    * Format a size passing a size in octet
+    * @param int $number
+    * @return string
+    */
+   public function fileSizeFilter($number)
+   {
+      return Toolbox::getSize($number);
+   }
```

Comment:
I do not because if we need to manipulate the value (which happens on some hooks) is more easy if there are the raw value also the data in bytes are sent to the web dashboard and they handle it in diffent way. 

And as it says on the Yii best practices for the views... "views should mainly contain presentational code, such as HTML, and simple PHP code to traverse, format and render data;" ;)

Reply:
ok, let's try. Next step is to find how to drop  the |raw in many variables of the templates .. HTML code  generated for dropdowns, ...